### ISSUE TITLE
Index format navigation

### ISSUE THREAD

**Issue by kenricqq** [mdn/content#17577]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/Reference

### What specific section or headline is this issue about?

index

### What information was incorrect, unhelpful, or incomplete?

When I initially approached the index trying to find something, I was perplexed in navigating the items. Control F could be an option, but not viable when one forgets the code. The idea of sorting by alphabet is great, but a suggestion for better navigation and readability is making the items horizontal, making them grouped and more centralized to a viewer.

### What did you expect to see?

ex.

A
 -  lorem      - lorem      - lorem      - lorem
 -  lorem      - lorem      - lorem
 
B
 -  lorem      - lorem      - lorem      - lorem
 -  lorem      - lorem      - lorem      - lorem

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/reference`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/Reference
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/reference/index.md
* Last commit: https://github.com/mdn/content/commit/a8bc68c6d7c25e8118e305948bf0857459eb7f16
* Document last modified: 2022-06-16T04:11:41.000Z

</details>

**Comment by wbamberg** [comment]
Yeah, I agree that would be better. This list is generated by a macro (https://github.com/mdn/yari/blob/main/kumascript/macros/CSS_Ref.ejs), so I'm moving his issue to the platform repo.
